ui/src/app/app-routing.module.ts

```diff
@@ -361,13 +391,9 @@ const routes: Routes = [
             path: 'institution/edit/:institutionId',
             component: AdminInstitutionEditComponent,
             data: { title: 'Institution Admin'},
-          }]
+          }
+        ]
       },
-      {
-        path: '**',
```

Comment:
What's a good way for me to check that all of the previously-wildcard routes are now accounted for?

Reply:
Cross reference `app-routing.tsx`